partitura/io/importkern.py

```diff
@@ -177,7 +177,7 @@ def _handle_ties(self):
                         self.tie_dict["open"].pop(index)
                         self._handle_ties()
             else:
-                for (oid, cid) in list(
+                for oid, cid in list(
```

Comment:
Looking at this line I wonder why is a list if the `zip` already returns an iterable (otherwise, this look would be even slower).

Reply:
Seems unnecessary to me too, should we change it?

Reply:
No, I think it is not a real issue, just something to keep in mind for the future.